### ISSUE TITLE
Support Google Cloud Storage as an alternative to AWS S3

### ISSUE THREAD

**Issue by TheBitFish** [transferwise/pipelinewise#677]
## Description
I'd like to have support for GCP added to Pipelinewise as my company doesn't use AWS.

## Acceptance
Allow Pipelinewise to be configured to use a GCS bucket as its external stage when loading data into Snowflake.

## Alternatives
* Local storage on the instance running Pipelinewise, undesirable because it's ephemeral
* Local storage on a mounted disk
* Work around by creating an S3 bucket, undesirable, adds another platform to our stack

## Additional context
Pipeline wise uses an external stage on S3 to load data into Snowflake. Support of other storage providers would add flexibility.

**Comment by TheBitFish** [comment]
If there are not any major flaws in this idea I'm happy to look into implementing it myself. lmk.

**Comment by Samira-El** [comment]
Hey Ryan,

This is a great idea, we always welcome any idea to make PipelineWise flexible and satisfy as many users as possible.

Currently, we unfortunately don't have capacity to work on this nor is it a priority given that we use AWS at Wise(formerly TransferWise). So it would be great if you could send a PR our way, we would gladly review it.

**Comment by TheBitFish** [comment]
@Samira-El Absolutely, just wanted to make sure there were no concerns or objections before I looked into it. I'll see what I can do!

**Comment by judahrand** [comment]
@TheBitFish This PR might be useful to you? https://github.com/transferwise/pipelinewise/pull/901
